svcb.go

```diff
@@ -235,15 +237,26 @@ type SVCBKeyValue interface {
 }
 
 // SVCBMandatory pair adds to required keys that must be interpreted for the RR
-// to be functional.
+// to be functional. If ignored, the whole RRSet must be ignored.
+// "port" and "no-default-alpn" are mandatory by default if present,
+// so they shouldn't be included here.
+//
+// It is incumbent upon the user of this library to reject the RRSet if
+// or avoid constructing such an RRSet that:
+// - "mandatory" is included as one of the keys of mandatory
```

Comment:
if you want this to make a list in godoc you need empty lines before and after. Maybe just make this more prose?

Reply:
I can't turn this list into a paragraph; it would be unreadable.